Orange/widgets/regression/owsgdregression.py

```diff
@@ -139,7 +144,8 @@ def set_data(self, data):
             self.apply()
 
     def apply(self):
-        loss = ["squared_loss", "huber", "epsilon_insensitive", "squared_epsilon_insensitive"][self.loss_function]
+        loss = ["squared_loss", "huber", "epsilon_insensitive",
+                "squared_epsilon_insensitive"][self.loss_function]
         penalty = ["l1", "l2", "elasticnet"][self.penalty_type]
         learning_rate = ["invscaling", "constant"][self.learning_rate]
```

Comment:
This is not really your fault, but I find these strings really far away from [their presentation counterparts](https://github.com/biolab/orange3/pull/1107/files#diff-8694470c8acf1a953aef01d51dc21efeR48). It's not good to have repeated code maintained at multiple places. Especially when it's simple to fix.